src/main/java/io/github/mivek/parser/AbstractParser.java

```diff
@@ -109,7 +97,7 @@ protected WeatherCondition parseWeatherCondition(final String weatherPart) {
      * @param pWeatherCode The weather code.
      * @param pTime        the string to parse.
      */
-    protected void parseDeliveryTime(final AbstractWeatherCode pWeatherCode, final String pTime) {
+    void parseDeliveryTime(final AbstractWeatherCode pWeatherCode, final String pTime) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/mivek/MetarParser/pullRequest?prid=4593741)